k8s-do/k8s/ingress-tls/README.md

```diff
@@ -0,0 +1,73 @@
+# ingress-tls
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.6.1](https://img.shields.io/badge/AppVersion-v1.6.1-informational?style=flat-square)
+
+ingress-tls installs and configures cert-manager, an ingress controller and a Let's Encrypt Issuer for automating TLS Certificates on AsyncAPI K8s services.
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.jetstack.io | cert-manager | 1.6.1 |
+| https://kubernetes.github.io/ingress-nginx | ingress-nginx | 4.0.16 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| cert-manager.installCRDs | bool | `true` |  |
+| certificates.issuer.acme.email | string | `"info@asyncapi.io"` |  |
+| certificates.issuer.acme.secret | string | `"letsencrypt-prod"` |  |
```

Comment:
That value isn't used, but I see `.certificates.issuer.acme.tokenSecretDestination`. Should it be that?

Reply:
THis is now fixed. good catch!